### ISSUE TITLE
Android app has No static method AnimatedContent issue while using voyager transitions 

### ISSUE THREAD

**Issue by sunildhiman90** [adrielcafe/voyager#151]
Hi, if we use voyager transitions, then while using SlideTransitions, we gets following issue:

```
java.lang.NoSuchMethodError: No static method AnimatedContent(Ljava/lang/Object;Landroidx/compose/ui/Modifier;Lkotlin/jvm/functions/Function1;Landroidx/compose/ui/Alignment;Ljava/lang/String;Lkotlin/jvm/functions/Function4;Landroidx/compose/runtime/Composer;II)V in class Landroidx/compose/animation/AnimatedContentKt; or its super classes (declaration of 'androidx.compose.animation.AnimatedContentKt' appears in /data/app/~~uvpuGkDwHhiCxB5RhweCgQ==/com.myapplication.MyApplication-Gmp19aONK-S9FmvOuDVCgg==/base.apk)
                 	at cafe.adriel.voyager.transitions.ScreenTransitionKt.ScreenTransition(ScreenTransition.kt:46)
                 	at cafe.adriel.voyager.transitions.SlideTransitionKt.SlideTransition(SlideTransition.kt:31)
                 	at ComposableSingletons$TabContentKt$lambda-1$1.invoke(TabContent.kt:33)
                 	at ComposableSingletons$TabContentKt$lambda-1$1.invoke(TabContent.kt:32)
```
its working fine for other platforms, but on android we have this issue.
Is there any workaround available for this?

**Affected platforms**
Android

**Versions**
Kotlin version*: 1.8.20
Compose Multiplatform version*: 1.4.0
voyager: 1.0.0-rc06

**To Reproduce**
I have created a reproducer for this issue, please checkout this repo:
https://github.com/sunildhiman90/compose-multiplatform-template-with-navigation
and choose the branch **with-voyager-navigation**
Try to build for androidApp.
It should reproduce this bug

**Comment by Syer10** [comment]
Make sure you are implementing Compose Animation and that your Jetpack Compose version is not using the 1.5 alphas

**Comment by sunildhiman90** [comment]
Thanks @Syer10 , i was missing the compose animation dependency in commonMain source set, it worked now.

PS: But i think it should be mentioned in documentation that if we want to use voyager navigation, then we should add compose animation dependency as well. thnx

**Comment by jakoss** [comment]
@Syer10 Is there a plan to support the 1.5 of compose animation? It's in beta now so the API should be stable

**Comment by Syer10** [comment]
It will likely only be supported when its fully released. Voyager uses Compose Multiplatform, which doesnt support Jetpack Compose alphas and betas.

**Comment by jakoss** [comment]
@Syer10 Nice to know, thanks for the info! :)

**Comment by 7kashif** [comment]
java.lang.NoSuchMethodError: No static method ModalBottomSheetLayout-BzaUkTc(Lkotlin/jvm/functions/Function3;Landroidx/compose/ui/Modifier;Landroidx/compose/material/ModalBottomSheetState;Landroidx/compose/ui/graphics/Shape;FJJJLkotlin/jvm/functions/Function2;Landroidx/compose/runtime/Composer;II)V in class Landroidx/compose/material/ModalBottomSheetKt; or its super classes (declaration of 'androidx.compose.material.ModalBottomSheetKt' appears in /data/app/~~9VpmHhRmwrZmqa_WNbIn_Q==/com.markaz.app.qa-nncJogGQRKKeRRBhTRBn9A==/base.apk)
                                                                                                    	at cafe.adriel.voyager.navigator.bottomSheet.BottomSheetNavigatorKt$BottomSheetNavigator$1$2.invoke(BottomSheetNavigator.kt:77)
                                                                                                    	at cafe.adriel.voyager.navigator.bottomSheet.BottomSheetNavigatorKt$BottomSheetNavigator$1$2.invoke(BottomSheetNavigator.kt:76)
                                                                                                    	at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:108)
                                                                                                    	at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:35)
                                                                                                    	
                                                                                                    	
I am also facing the similar issue with ModelBottomSheet, can you guyz please help me out on it, I have tried a lots of the things but nothing worked out

**Comment by igorbiscanin** [comment]
@adrielcafe Any updates about this?

**Comment by CodeQuacker** [comment]
about animatedContent look for my comment here #120

**Comment by adrielcafe** [comment]
Fixed on [1.0.0-rc07](https://github.com/adrielcafe/voyager/releases/tag/1.0.0-rc07).
